tests/parsers/test.properties.js

```diff
@@ -0,0 +1,48 @@
+import Linter from 'linter';
+import PropertiesParser from 'parsers/properties';
+
+describe('PropertiesParser', () => {
+  it('should parse valid .properties file correctly', () => {
+    const addonLinter = new Linter({ _: ['bar'] });
+    const propertiesParser = new PropertiesParser(`
+      foo=bar
+      abc.def = xyz
+      #Comment
+      fast=ball
+
+      #Empty spaces
+
+      this.is.a =test
+      overwrite=foo
+      overwrite=bar`, addonLinter.collector);
+
+    propertiesParser.parse();
+
+    expect(propertiesParser.isValid).toEqual(true);
+    expect(propertiesParser.parsedData).toEqual({
+      foo: 'bar',
+      'abc.def': 'xyz',
+      fast: 'ball',
+      'this.is.a': 'test',
+      overwrite: 'bar',
+    });
+  });
+
+  it('should ignore invalid entities', () => {
+    const addonLinter = new Linter({ _: ['bar'] });
+    const propertiesParser = new PropertiesParser(`
+        this should be ignored.
+        foo=
+        bar
+        abc.def=x
+        yz`, addonLinter.collector);
```

Comment:
Could we add another string that comprises of three lines? Just to make sure we don't break the implementation in the future.

Reply:
Will do :)